mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java

```diff
@@ -35,24 +36,31 @@ public class ProjectJpaRepository implements ProjectRepository {
   private final ProjectPreviewRepository projectPreviewRepo;
```

Comment:
As the name indicates, I would not expect the project preview in the domain. QbicProjectPreviewRepository sounds wrong. Is this really a concept of the domain or rather a simplified data container for the view?